### PR TITLE
chore: Bump version to 0.1.0-alpha.8-pre

### PR DESCRIPTION
**Pull request by heeckhau**

### CODE REVIEW

File: crates/tests-integration/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tests-integration"
-version = "0.0.0"
```
Comment (th4s) [inline crates/tests-integration/Cargo.toml:3]: Is this a mistake?

File: crates/tls/server-fixture/Cargo.toml [hunk 2]
```diff
@@ -5,7 +5,7 @@ description = "Fixtures for testing"
 keywords = ["tls", "fixture"]
 categories = ["cryptography"]
 license = "MIT OR Apache-2.0"
-version = "0.0.0"
```
Comment (th4s) [inline crates/tls/server-fixture/Cargo.toml:8]: same

**Review by th4s — COMMENTED** [review]
Do we want to version  these two?

File: crates/tls/server-fixture/Cargo.toml [hunk 3]
```diff
@@ -5,7 +5,7 @@ description = "Fixtures for testing"
...
+version = "0.1.0-alpha.8-pre"
```
Comment (th4s) [inline crates/tls/server-fixture/Cargo.toml:8]: same

File: crates/tests-integration/Cargo.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
+version = "0.1.0-alpha.8-pre"
```
Comment (th4s) [inline crates/tests-integration/Cargo.toml:3]: Is this a mistake?

Comment (heeckhau) [reply crates/tests-integration/Cargo.toml:3]: @th4s Thanks for catching this. I found one more in the examples crate

**Pull request merged**